src/components/workspace/side-content/AutograderCard.tsx

```diff
@@ -47,45 +24,39 @@ class AutograderCard extends React.Component<AutograderCardProps, {}> {
       }
     };
 
+    const isCorrect =
```

Comment:
Please rewrite this ternary portion and make things more readable.

Reply:
Done. I think it should be better now?

Reply:
We can improve it further actually! However, lets refer to my latest comment for now!